com.github.vinser.burnfix.yml

```diff
@@ -0,0 +1,37 @@
+app-id: com.github.vinser.burnfix
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.golang
+command: burnfix
+
+finish-args:
+    - --share=ipc # Share IPC namespace with the host (necessary for X11).
+    - --socket=x11
+    - --device=dri # OpenGL rendering support.
+
+    # Needed to support desktop notifications.
+    # - --talk-name=org.freedesktop.Notifications
```

Comment:
Where are you using this? In main.go I don't see anything related to notifications or fyne.notifications.